src/main/java/proto/Pose.proto

```diff
@@ -0,0 +1,20 @@
+/*
+ * This proto format is used along with the delimited methods to
+ * store message size for easy decoding/encoding.
+ */
+syntax = "proto3";
+
+package formats;
+option java_package = "proto";
+option java_outer_classname = "SensorData";
+
+message Pose{
+  // Position is in WGS84 latitude, longitude, altitude.
+  double gps_timestamp = 1; // Time in seconds.
+  double lat = 2;  // Degrees in [-90, 90].
+  double lng = 3;  // Degrees in [-180,180].
```

Comment:
Why allow multiple representations for the same location? Usually we would have these as half-open intervals.

Reply:
I copied these degree values from a previous comment by Jie I'm not totally sure the reasoning behind this interval.